pkg/auth/base.go

```diff
@@ -16,7 +16,7 @@ var (
 // without the required ARN.
 var statementByName = map[string]model.Statement{
 	"AllAccess": {
-		Action: []string{"fs:*", "auth:*", "ci:*", "retention:*", "branches:*"},
```

Comment:
In some places it's `pull`, in others it's `pr`. 
Any logic behind it?

(It might searching in the repo difficult.)

Reply:
`pr` prefix for permission is according to the PRD specification.
Where did you see we use pull?